src/main/steps/c100-rebuild/child-details/childDetailPostController.ts

```diff
@@ -76,7 +76,7 @@ export default class ChildDetailsPostController {
 
     req.session.userCase.cd_children = updatePartyDetails(
       {
-        ...getPartyDetails(childId, req.session.userCase.cd_children!),
+        ...getPartyDetails(childId, req.session.userCase.cd_children),
```

Comment:
cd-children is an optional prop. Are you sure cd_children value does not come empty?

Reply:
Lint is working fine on that.